domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/education/EducationLevel.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.justice.digital.hmpps.domain.learningandworkprogress.education
+
+/**
+ * An enumeration of Education Levels
```

Comment:
Appears to be a new enum, but has just been moved out of `PreviousQualifications.kt` into it's own source file.